vllm/config/__init__.py

```diff
@@ -3196,6 +3196,103 @@ def _parse_collect_detailed_traces(self):
             list[DetailedTraceModules],
             self.collect_detailed_traces[0].split(","))
 
+ECProducer = Literal["ec_producer"]
+ECConsumer = Literal["ec_consumer"]
+ECRole = Literal[ECProducer, ECConsumer]
+@config
+@dataclass
+class ECTransferConfig:
```

Comment:
If this config is to stay (pending @NickLucche's comment), it should live in its own file/with other connectors. Configs are no longer defined in `__init__.py`